server/notifications/index.ts

```diff
@@ -0,0 +1,35 @@
+import User from './User';
+import Publisher from './Publisher';
+import { PostManager } from '../db/PostManager';
+const users = {};
+export const publisher = new Publisher({});
```

Comment:
I have concerns regarding these variables. The functions that use them are hoisted but do not check if they exist.
I guess in most cases it shouldn't create problems. But it might cause unpredictable errors.